Meridian59/Data/Models/AdminData/AdminObject.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.ComponentModel;
+using Meridian59.Data.Lists;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    /// <summary>
+    /// Object that represents a displayed object in the Administration UI
+    /// </summary>
+    public class AdminObject : INotifyPropertyChanged
+    {
+        /// <summary>
+        /// The Object's Properties
+        /// </summary>
+        public BaseList<AdminObjectProperty> Properties { get; private set; } 
+
+        /// <summary>
+        /// Kod Class
+        /// </summary>
+        public string ClassName { get; set; }
+        /// <summary>
+        /// Object Number
+        /// </summary>
+        public int ObjectNumber { get; set; }
+
+        public AdminObject(string classname, int objectnumber, BaseList<AdminObjectProperty> props = null)
+        {
+            ClassName = classname;
+            ObjectNumber = objectnumber;
+            if (props != null)
+            {
+                Properties = props;
+            }
+            else
+            {
+                Properties = new BaseList<AdminObjectProperty>();
```

Comment:
I prefer initialising collection instances once and define them readonly, to make sure they are correctly cleared/etc. rather than overwritten with a new instance (important for databinding!).

To achieve this, you should add the following field declaration to your class:
>> protected readonly BaseList<AdminObjectProperty> properties = new BaseList<..>();

Then return this instance in the getter of your:
>> public BaseList<AdminObjectProperty> Properties { get { return properties; } } 

In your constructor, you would then have something like:
>> public AdminObject(string classname, int objectnumber, IEnumerable<AdminObjectProperty> props = null)

And in its implementation, you add the items to your own collection:
>> for(.... in props) ..[addtoownpropertieslist]